media-gfx/librecad/files/librecad-2.1.3-CVE-2021-21898.patch

```diff
@@ -0,0 +1,371 @@
+From ba3fa95648bef948e008dfbdd31a4d21badd71f0 Mon Sep 17 00:00:00 2001
```

Comment:
Please add a link to where you got it from and also any relevant Gentoo bugs.

Reply:
fixed